client/devlxd_images.go

```diff
@@ -0,0 +1,19 @@
+package lxd
+
+import (
+	"fmt"
+
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/version"
+)
+
+// ExportImage exports the image with a given fingerprint from the host's LXD.
+// Note that only cached and public images can be exported.
+func (r *ProtocolDevLXD) ExportImage(fingerprint string, req ImageFileRequest) (*ImageFileResponse, error) {
```

Comment:
is ExportImage the right name here? What about `GetImageFile` which seems to align more with the existing client package in terms of function signature.